README.md

```diff
@@ -1 +1,54 @@
-# designing_microservices_course
\ No newline at end of file
+# Designing microservices course
+This repository is intended for demonstration purposes for students during the "Basics of microservices architecture 
```

Comment:
maybe: This repository contains demo project which can be used by ITMO students enrolled in "Microservices architecture basics" course (taught in 2019 autumn term of ${your_faculty_or_department_name})

I think that info in parentheses can be dropped.